### ISSUE TITLE
Support __declspec(align(n))

### ISSUE THREAD

**Issue by fafalone** [twinbasic/twinbasic#1308]
**Is your feature request related to a problem? Please describe.**
Some UDTs for data use features like CACHE_ALIGN that pad them to multiples of the specified alignment number (in the case of cache align, usually 32 or 64 bytes). This is different from `#pragma pack(n)`, which is now supported via the `[ PackingAlignment(n) ]` attribute. If tB supports that, it ought to support this too if it's not too much trouble. 

**Describe the solution you'd like**
.NET did us no favors by calling pragma pack 'PackingAlignment' to set the standard, so I'm not sure what a good attribute name would be... StructureAlignment? DataAlignment? 

**Describe alternatives you've considered**
Manual alignment.

**Additional context**
I've seen it applied to individual members in addition to entire structures; so that might complicate it a bit. I don't know how common that is.

**Comment by WaynePhillipsEA** [comment]
What about calling it `MemberAlignment`?  When applied to a field member, it applies only to that field.   When applied to a UDT itself, it applies to any member (or variable declaration) that uses that UDT.

**Comment by fafalone** [comment]
Makes sense, I like that better.

For where/how it applies, I'd presume it should follow the rules of C, as 99% of usage is probably going to be using C stuff in tB.

**Comment by WaynePhillipsEA** [comment]
Yeah, that is how it applies there... I was just trying to convince myself that `MemberAlignment` was a suitable candidate
